src/policy-store/userdata.js

```diff
@@ -0,0 +1,19 @@
+const POLICY_KEY = "enlearnPolicy";
```

Comment:
I'm trying to understand how we should think of the difference between `userdata.js` and `create-policy-store.js`. The name `userdata.js` also makes me think that the exported object should be some sort of data store or DTO, but it's essentially providing an interface to the userData.